server/src/main/java/com/google/fhir/proxy/BearerAuthorizationInterceptor.java

```diff
@@ -170,7 +174,13 @@ DecodedJWT decodeAndVerifyBearerToken(String authHeader) {
           AuthenticationException.class);
     }
     String bearerToken = authHeader.substring(BEARER_PREFIX.length());
-    DecodedJWT jwt = JWT.decode(bearerToken);
+    DecodedJWT jwt = null;
+    try {
+      jwt = JWT.decode(bearerToken);
+    } catch (JWTDecodeException e) {
+      ExceptionUtil.throwRuntimeExceptionAndLog(
+          logger, "Failed to decode JWT: " + e.getMessage(), e, AuthenticationException.class);
```

Comment:
There are [fixed](https://github.com/google/fhir-access-proxy/commit/eec20d1a52e8136ea414237bfea979eeefddeb2e) upstream; let's make sure we fix the dependency instead of forking.

Reply:
Noted 👍🏾